docs/api.md

```diff
@@ -1342,6 +1237,125 @@ Name of the current workspace.
 URL of the current plugin engine.
 
 
+## Internal plugins
+
+Besides the default ImJoy api, we provide a set of internally supported plugins which can be used directly. These plugins will be loaded only if the plugin is requested by other plugin via `api.getPlugin(...)`.
```

Comment:
These plugins will be loaded only of the plugin is requested by another plugin via ...